.github/workflows/llm-harness-evaluation.yml

```diff
@@ -122,6 +124,7 @@ jobs:
       - name: Install dependencies
         shell: bash
         run: |
+          set -e
```

Comment:
what's this for?

Reply:
Exit script on any error

Reply:
why? does it mean skipping this step if any failure occurs?